### PR TITLE
[feat] adds envio as supported data indexer

### PR DESCRIPTION
**Pull request by SvenMuller95**
## What is the purpose of the change?

add new content based on https://github.com/sei-protocol/sei-docs/pull/88

fixes merge conflict https://github.com/sei-protocol/sei-docs/pull/88

### CODE REVIEW

**Comment by SvenMuller95** [comment]
@cordt-sei created this PR in favour of https://github.com/sei-protocol/sei-docs/pull/88

**Review by cordt-sei — COMMENTED** [review]
Looks good, please note we will be migrating these to a whole new expanded section for third party/community contributions in the near future.

**Review by cordt-sei — APPROVED** [review]

File: pages/dev-ecosystem-providers/indexers/indexers.mdx [hunk 1]
```diff
@@ -20,3 +20,9 @@ Allows for querying blockchain data using GraphQL.
 SubQuery is a fast, flexible, and reliable open-source data decentralised infrastructure network, providing both RPC and indexed data to consumers around the world.
 
 - [SubQuery Docs](https://academy.subquery.network/indexer/quickstart/quickstart_chains/cosmos-sei.html)
+
+## **Envio (EVM only)**
+
+Envio is a modern, multi-chain EVM blockchain indexing framework speed-optimized for querying real-time and historical data.
```
Comment (ghost) [inline pages/dev-ecosystem-providers/indexers/indexers.mdx]:
```suggestion
Envio is a modern, multi-chain EVM blockchain indexing framework, speed-optimized for querying real-time and historical data. Learn more in the [official Envio Docs](https://docs.envio.dev/docs/contract-import).
```

File: pages/dev-ecosystem-providers/indexers/indexers.mdx [hunk 2]
```diff
@@ -20,3 +20,9 @@ Allows for querying blockchain data using GraphQL.
...
+
+   - [Envio Docs](https://docs.envio.dev/docs/contract-import)
```
Comment (ghost) [inline pages/dev-ecosystem-providers/indexers/indexers.mdx]:
```suggestion
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by SvenMuller95** [comment]
Hey @cordt-sei @seiyan-writer , is this PR all good to go ? Please let me know 🙏 

Would it also be possible to change Envio from 'infrastructure' to 'data & analytics" label / category on the ecosystem?
Screenshot 2024-10-23 at 16 48 08

**Comment by cordt-sei** [comment]
It looks like this page is being dynamically generated from somewhere else now.

@partyparrotgreg where can @SvenMuller95 make this PR now?

**Pull request closed without merging**